README.md

```diff
@@ -1,7 +1,7 @@
 ## Installation
 
 ```
-npm install --save @slite/quill-mardown-delta
+npm install --save @sliteteam/quill-delta-markdown
```

Comment:
It should be `@slite/quill-delta-markdown`
https://www.npmjs.com/package/@slite/quill-delta-markdown